beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -241,6 +243,7 @@ def __init__(self, qspec: TFQuantizerSpec):
         self.per_channel = qspec.per_channel
         self.narrow_range = qspec.narrow_range
         self.signedness_to_force = qspec.signedness_to_force
+        self._is_applied_saturation_fix = qspec.apply_saturation_fix
```

Comment:
Please, consider using more shorter and clear naming for the Quantization operation attribute. For example,

self._is_applied_saturation_fix- > self.half_range

half_range is a bool parameter that indicates that half of the quantization range will be used. If half_range = True and  num_bits = 8 then the range [-64, 63] will be used for quantization instead of [-128, 127]

```suggestion
        self.half_range = qspec.apply_saturation_fix
```

Reply:
Should this attribute really be exposed to the outside world? We are ultimately only performing workarounds here, not adding a feature.

Reply:
This attribute will be saved in the model and any user can see this attribute as an attribute of the quantization operation. Anyway, I agree to leave `apply_saturation_fix` but this name will use everywhere.

```suggestion
        self.apply_saturation_fix= qspec.apply_saturation_fix
```

Reply:
If @vshampor agrees I am ready to change the name to `half_range`. 
I feel like it is more convenient, especially, if we have `narrow_range ` attribute that almost have the same idea behind

Reply:
Updated according the discussion
@alexsu52 please take a look